src/GeositeFramework/js/Plugin.js

```diff
@@ -401,6 +405,10 @@ require(['use!Geosite',
             view.$uiContainer = $uiContainer;
 
             $uiContainer
+                // Call `hibernate` method on plugin when the eye icon is clicked
+                .find('.plugin-eye').on('click', function () {
+                    model.hibernate();
+                }).end()
```

Comment:
It looks like the `.deselect` method called in the `.plugin-close` case also removes the sidebar rather than just having it remove the layers. If we do want it the eye-icon button to close the sidebar too, I can have it use `.deselect` method after `.hibernate`.

I think either way I'll cut the `.plugin-close` case, as that's no longer a button used in the app.